docs/internals/controls-package.md

```diff
@@ -136,8 +396,8 @@ The Constant block outputs a constant signal. S_out = Cte
 ### Endpoint
 
 The Endpoint is a special type of block that is used to represent a plant.
-For now, only DC motors with an encoder are supported as an endpoint in the control package.
-The _motor_name_ attribute is unused for now, and one should pass a Controllable interface when creating the loop.
+For now, only DC motors with an encoder, and a base with a linear and angular movement sensor are supported as an endpoint in the control package.
+The _motor_name_ attribute should be changed to _base_name_ when using a base.
```

Comment:
We try to speak to the user as much as possible because otherwise they may think we will change it. This makes it clear they need to perform the action
```suggestion
You should change the _motor_name_ attribute to _base_name_ when using a base.
```